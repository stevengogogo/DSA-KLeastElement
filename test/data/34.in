10 50000
-3288 9550 -1820 3163 -9860 -2398 -7910 6820 -399 -2397
Delete 5
Query 7 9 1
Insert 8 -5019
Delete 1
Delete 3
Delete 6
Insert 7 1571
Query 8 8 1
Query 6 6 1
Query 2 3 1
Reverse 7 7
Reverse 3 6
Delete 8
Reverse 1 4
Delete 2
Query 4 5 2
Reverse 4 6
Delete 1
Query 5 5 1
Delete 3
Query 2 3 1
Delete 3
Reverse 1 3
Reverse 1 2
Reverse 2 2
Insert 2 -4340
Insert 1 1620
Query 5 5 1
Delete 5
Reverse 1 1
Insert 5 -3033
Query 1 5 1
Insert 5 -5839
Insert 1 4182
Reverse 3 4
Query 7 7 1
Delete 4
Reverse 6 6
Delete 2
Delete 4
Reverse 4 4
Reverse 3 4
Delete 2
Insert 4 5125
Reverse 2 4
Reverse 2 3
Query 3 4 2
Query 2 3 2
Insert 2 744
Delete 1
Reverse 4 4
Delete 3
Query 1 3 1
Reverse 2 3
Delete 2
Insert 2 5318
Delete 2
Reverse 2 2
Insert 3 9433
Insert 1 5077
Insert 2 -1249
Insert 3 -342
Insert 5 -68
Query 2 6 2
Delete 7
Insert 3 -9368
Delete 6
Delete 6
Query 1 4 1
Reverse 3 5
Query 4 4 1
Insert 6 5299
Delete 5
Reverse 1 1
Query 4 5 1
Insert 1 -8997
Delete 2
Insert 5 2271
Insert 4 3288
Insert 6 373
Delete 6
Reverse 1 4
Insert 4 556
Insert 9 -1798
Reverse 3 9
Reverse 3 8
Query 4 7 2
Query 3 4 2
Query 5 6 2
Insert 4 -8655
Insert 1 -7476
Query 6 8 3
Query 3 6 1
Insert 7 6078
Query 5 7 1
Delete 1
Reverse 1 11
Query 3 11 4
Insert 7 4426
Query 3 8 1
Insert 9 -2455
Query 10 11 2
Insert 10 7772
Delete 4
Insert 9 582
Query 14 14 1
Delete 14
Reverse 9 10
Query 12 13 2
Delete 1
Reverse 6 9
Query 6 10 4
Insert 7 -8245
Delete 4
Insert 8 -8119
Reverse 9 13
Query 8 13 6
Reverse 10 10
Query 12 12 1
Reverse 3 4
Delete 2
Reverse 4 8
Delete 12
Insert 2 336
Delete 4
Query 3 8 3
Insert 7 -8561
Query 1 5 3
Insert 2 -3118
Insert 8 -3005
Query 1 14 11
Reverse 2 8
Query 5 9 2
Delete 1
Query 5 11 7
Reverse 3 13
Reverse 8 12
Delete 5
Query 1 2 1
Query 2 10 5
Delete 9
Insert 4 8646
Query 4 12 5
Delete 1
Reverse 2 6
Reverse 7 9
Reverse 11 11
Reverse 3 7
Insert 9 726
Delete 9
Query 3 5 2
Reverse 7 8
Delete 5
Delete 4
Query 4 6 3
Query 1 9 2
Insert 10 -9886
Reverse 8 8
Query 7 10 2
Query 4 10 1
Query 8 9 2
Query 5 9 4
Insert 2 4501
Delete 7
Insert 10 -5447
Reverse 8 10
Query 2 6 3
Query 11 11 1
Delete 1
Reverse 3 9
Reverse 6 7
Query 4 8 2
Insert 8 3582
Insert 9 -3313
Query 8 12 2
Delete 10
Insert 3 -6529
Delete 3
Query 8 8 1
Delete 1
Reverse 10 10
Query 2 6 1
Query 3 6 2
Reverse 5 10
Query 1 6 4
Reverse 10 10
Delete 2
Insert 4 -3239
Insert 6 9849
Reverse 6 11
Delete 3
Insert 3 6205
Query 11 11 1
Delete 5
Delete 6
Delete 6
Insert 3 5285
Delete 4
Reverse 5 5
Delete 6
Reverse 6 7
Reverse 5 7
Delete 3
Insert 7 -2774
Insert 4 -3635
Reverse 8 8
Reverse 4 7
Delete 5
Delete 3
Reverse 6 6
Insert 5 -4081
Reverse 6 6
Reverse 7 7
Reverse 4 5
Delete 3
Reverse 3 4
Reverse 5 6
Delete 4
Insert 4 -7267
Delete 4
Insert 1 3232
Delete 3
Delete 4
Query 2 4 2
Reverse 1 1
Reverse 4 4
Query 1 1 1
Query 3 4 2
Reverse 2 3
Delete 1
Query 3 3 1
Reverse 2 2
Insert 2 2538
Reverse 2 3
Delete 1
Insert 1 4829
Delete 3
Delete 2
Reverse 2 2
Reverse 2 2
Delete 2
Delete 1
