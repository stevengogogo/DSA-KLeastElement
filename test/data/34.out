-2397
-2397
-399
-2398
-2398
-7910
-2398
-1820
-7910
-3033
5125
5125
-7910
-1249
-9368
-342
-342
-342
556
2271
2271
-8997
-8997
556
-8997
556
3288
744
582
744
-8997
582
336
582
-3118
4426
-3005
336
-3118
-3118
4426
-8561
-8561
-9886
-8119
744
-3118
-9886
-8119
-8655
3582
-8561
-5447
-8119
9849
-3635
3232
-3118
-3635
